src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -1,4 +1,15 @@
 package com.spartronics4915.frc2020;
 
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.I2C.Port;
+
 public final class Constants {
-}
+    public static final int kBeamSensorUpID = -1;
```

Comment:
Please add all of these devices and IDs in the [2020 control map](https://docs.google.com/spreadsheets/d/14O3Dj1Bl6tqiHFrCJ7wyVRDOqS5EZ11p4q9j2rYS-3Q/edit#gid=0).

Reply:
That's a placeholder - we haven't been given this ID yet.